src/CMakeLists.txt

```diff
@@ -21,13 +21,14 @@ install(TARGETS smfc
   EXPORT smf-export
   DESTINATION ${CMAKE_INSTALL_BINDIR})
 
-# TODO: rpc_generated.h should not be checked into git. instead, it should be
-# generated at build-time and installed/packaged explicitly.
 smfc_gen(
   CPP
   TARGET_NAME raw_rpc_gen
   OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/include/smf
   SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/core/rpc.fbs)
+install(
```

Comment:
@noahdesu  this was the func missing on issue #317

Reply:
Yep. Looks like you are tackling some/all of the installation stuff I didn't get around to in the cmake-ng PR. I'll build tickets for the remainder of the todo's there.

Reply:
yeah, i just wanted to fix the install() issues at least for redpanda on a speedy schedule. thanks for following up tho ! :) :+1:  :100: